.github/ISSUE_TEMPLATE/z_bug_report.yml

```diff
@@ -0,0 +1,91 @@
+name: Report an issue with Home Assistant Core (Test)
```

Comment:
```suggestion
name: Report an issue with Home Assistant Core
```
test?

Reply:
Test, the original one is still here. As I don't know how this will affect the bug.
Because the file name start with `z_` it should appear at the bottom of the issue chooser, giving us a possibility to test that first.

While the old one still is displayed. I added the test marker to make it more clear from the issue chooser.